tests/base.py

```diff
@@ -47,6 +49,9 @@
 )
 test_app.testing = True
 
+DISCONNECTED_SIGNALS = {
+    contributor_added: [notify_added_contributor]
+}
```

Comment:
However, it would make sense to make this a class constant so that it can be configured for individual TestCases.